### PR TITLE
Logo changes and link logo to site

### PR DESCRIPTION
**Pull request by yhaspel**
* Logo changes
* Link logo in app to spacemesh.io site

### CODE REVIEW

File: app/screens/main/Main.js [hunk 1]
```diff
@@ -143,11 +145,17 @@ class Main extends Component<Props, State> {
         <SideBar src={sideBar} />
         <NavBar>
           <NavBarPart>
-            <Logo src={logo} />
+            <Logo src={logo} onClick={() => this.handleNavigation({ index: 6 })} />
             <NavLinksWrapper>
-              <NavBarLink onClick={() => this.handleNavigation({ index: 0 })} isActive={activeRouteIndex === 0}>MINING</NavBarLink>
-              <NavBarLink onClick={() => this.handleNavigation({ index: 1 })} isActive={activeRouteIndex === 1}>WALLET</NavBarLink>
-              <NavBarLink onClick={() => this.handleNavigation({ index: 2 })} isActive={activeRouteIndex === 2}>CONTACTS</NavBarLink>
+              <NavBarLink onClick={() => this.handleNavigation({ index: 0 })} isActive={activeRouteIndex === 0}>
```
Comment (IlyaVi) [inline app/screens/main/Main.js]: what the reason for changing this and following lines?

File: app/screens/main/Main.js [hunk 2]
```diff
@@ -143,11 +145,17 @@ class Main extends Component<Props, State> {
         <SideBar src={sideBar} />
         <NavBar>
           <NavBarPart>
-            <Logo src={logo} />
+            <Logo src={logo} onClick={() => this.handleNavigation({ index: 6 })} />
```
Comment (IlyaVi) [inline app/screens/main/Main.js]: can it be done like this: onClick={() => shell.openExternal('https://spacemesh.io')}
or should it be done by changing multiple lines of code and disrupting the existing logic (numeration order)?

Comment (yhaspel) [reply app/screens/main/Main.js]: I was using your existing structure. I'll change according to your comment.

Comment (IlyaVi) [reply app/screens/main/Main.js]: you didn't cause according to this logic icon nav should be with index 0

**Pull request closed without merging**